### PR TITLE
[HuongDoiTuong] Bai Tap 1

### PR DESCRIPTION
**Pull request by ThaoTranHD**
Bai tap ve thong tin Ho Gia Dinh
Screen Shot 2022-10-13 at 7 58 03 AM

### CODE REVIEW

File: HuongDoiTuong/BaiTap1.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+let nguoi1 = Nguoi(ten: "thao", tuoi: 18, ngheNghiep: "IT", soCMND: 01)
+let nguoi2 = Nguoi(ten: "Phat", tuoi: 18, ngheNghiep: "IT", soCMND: 02)
+let nguoi3 = Nguoi(ten: "Hung", tuoi: 20, ngheNghiep: "Free", soCMND: 03)
+let hoGD1 = HoGiaDinh(soNha: "215 PNX", cacThanhVien: [nguoi1, nguoi2])
+let hoGD2 = HoGiaDinh(soNha: "306 TCV", cacThanhVien: [nguoi3])
+let khuPho = KhuPho(tenKhuPho: "A", cacHo: [hoGD1, hoGD2])
+
+
```
Comment (PaxtonNguyen6495) [inline HuongDoiTuong/BaiTap1.playground/Contents.swift]: remove line

Comment (ThaoTranHD) [reply HuongDoiTuong/BaiTap1.playground/Contents.swift]: yes sir

File: HuongDoiTuong/BaiTap1.playground/contents.xcplayground [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (PaxtonNguyen6495) [inline HuongDoiTuong/BaiTap1.playground/contents.xcplayground]: remove irrelevant file

**Comment by PaxtonNguyen6495** [comment]
pending this feature

**Pull request closed without merging**